flowise-embed/src/features/menu/components/Menu.tsx

```diff
@@ -32,6 +32,18 @@ export const Menu = (props: MenuProps) => {
   const [modalPosition, setModalPosition] = createSignal<'top' | 'bottom'>('bottom');
   const [selectedChatId, setSelectedChatId] = createSignal<string | null>(null);
   const [selectedChatName, setSelectedChatName] = createSignal<string | null>(null);
+  const [activeChatId, setActiveChatId] = createSignal<string | null>(null);
+
+  let menuItemsRef: HTMLDivElement | undefined;
+  createEffect(() => {
+    const currentActiveChatId = activeChatId();
+    if (currentActiveChatId && menuItemsRef) {
+      const menuItems = menuItemsRef.getElementsByClassName('menu-history-item');
+      Array.from(menuItems).forEach((item) => {
```

Comment:
Nesse caso aqui, precisa do `Array.from`? Só um `menuItems.forEach()` não funcionava?

Reply:
Fica alertando o erro: 'Property 'forEach' does not exist on type 'HTMLCollectionOf<Element>'.